k8s/namespaces/toffee/kustomization.yaml

```diff
@@ -7,4 +7,5 @@ resources:
 - toffee-recipe-backend/toffee-azure-identity-binding.yaml
 - toffee-recipe-backend/toffee-azure-identity.yaml
 - toffee-frontend/toffee-frontend.yaml
-
+- toffee-recipe-backend/apple-azure-identity-binding.yaml
```

Comment:
this doesn't belong in the main kustomization it's only for sbox

Reply:
removed the base config.